pyproject.toml

```diff
@@ -74,6 +74,7 @@ dependencies    = [
     "voluptuous-serialize==2.6.0",
     "voluptuous-openapi==0.0.5",
     "yarl==1.16.0",
+    "webrtc-models==0.0.0b2",
```

Comment:
Would be good to get a non beta release of this requirement before the November release.

Reply:
Are you not a believer in [zerover](https://0ver.org/)?

Reply:
I don't mind 0. releases, but I think that the beta label means that the maintainer thinks it's not yet ready for common consumption.

Reply:
Will bump it to a stable version before the stable HA release